README.md

```diff
@@ -8,6 +8,11 @@ This repository contains all Java related sources of Rico. Clients for other lan
 
 ![Rico Logo](readme/rico_logo.png)
 
+![Hacktoberfest Logo](https://hacktoberfest.digitalocean.com/assets/HF-full-logo-b05d5eb32b3f3ecc9b2240526104cf4da3187b8b61963dd9042fdc2536e4a76c.svg)
+
+*This project joined [Hacktoberfest](https://hacktoberfest.digitalocean.com/)!
+Issues that can give you a good start are [have the label "Hacktoberfest"](https://github.com/rico-projects/rico/issues?q=is%3Aissue+is%3Aopen+label%3A%22Hacktoberfest%22) and maintainers are willing to help.*
```

Comment:
As far as I know it is "hacktoberfest" label and not "Hacktoberfest". Does this make a difference?

Reply:
No, it doesnt - but the issues in our repo are actually uppercase ;-)